Solutions/AliCloud/DataConnectors/AliCloud_API_FunctionApp.json

```diff
@@ -0,0 +1,121 @@
+		"query": "AliCloud\n | sort by TimeGenerated desc"
+	}
+],
+	"dataTypes": [{
+			"name": "AliCloud_CL",
+			"lastDataReceivedQuery": "AliCloud_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+		}
+	],
+	"connectivityCriterias": [{
+			"type": "IsConnectedQuery",
+			"value": [
+				"AliCloud_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+			]
+		}
+	],
+	"availability": {
+		"status": 1,
+		"isPreview": true
+	},
+	"permissions": {
+		"resourceProvider": [{
+				"provider": "Microsoft.OperationalInsights/workspaces",
+				"permissionsDisplayText": "read and write permissions on the workspace are required.",
+				"providerDisplayName": "Workspace",
+				"scope": "Workspace",
+				"requiredPermissions": {
+					"write": true,
+					"read": true,
+					"delete": true
+				}
+			},
+			{
+				"provider": "Microsoft.OperationalInsights/workspaces/sharedKeys",
+				"permissionsDisplayText": "read permissions to shared keys for the workspace are required. [See the documentation to learn more about workspace keys](https://docs.microsoft.com/azure/azure-monitor/platform/agent-windows#obtain-workspace-id-and-key).",
+				"providerDisplayName": "Keys",
+				"scope": "Workspace",
+				"requiredPermissions": {
+					"action": true
+				}
+			}
+		],
+		"customs": [{
+				"name": "Microsoft.Web/sites permissions",
+				"description": "Read and write permissions to Azure Functions to create a Function App is required. [See the documentation to learn more about Azure Functions](https://docs.microsoft.com/azure/azure-functions/)."
+			},
+			{
+				"name": "REST API Credentials/permissions",
+				"description": "**AliCloudAccessKeyId** and **AliCloudAccessKey** are required for making API calls."
+			}
+		]
+	},
+	"instructionSteps": [{
+			"title": "",
+			"description": ">**NOTE:** This connector uses Azure Functions to connect to the Azure Blob Storage API to pull logs into Microsoft Sentinel. This might result in additional costs for data ingestion and for storing data in Azure Blob Storage costs. Check the [Azure Functions pricing page](https://azure.microsoft.com/pricing/details/functions/) and [Azure Blob Storage pricing page](https://azure.microsoft.com/pricing/details/storage/blobs/) for details."
+		},
+		{
+			"title": "",
+			"description": ">**(Optional Step)** Securely store workspace and API authorization key(s) or token(s) in Azure Key Vault. Azure Key Vault provides a secure mechanism to store and retrieve key values. [Follow these instructions](https://docs.microsoft.com/azure/app-service/app-service-key-vault-references) to use Azure Key Vault with an Azure Function App."
+		},
+		{
+			"description": ">**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected [**AliCloud**](https://aka.ms/sentinel-AliCloud-parser) which is deployed with the Microsoft Sentinel Solution."
```

Comment:
The link " https://aka.ms/sentinel-AliCloud-parser "  redirecting to a different page below. Please cross verify the link.
![image](https://user-images.githubusercontent.com/105280229/170717159-b0c9d6c7-dc60-427c-abaa-d2017d558689.png)

Reply:
Hey @KateShapovalova, @VenkataPrathibha, updated the link and its working, can you please check. Thanks.

Reply:
@v-sabiraj, the link is working. Thanks.